<testsuite errors="0" failures="0" hostname="DESKTOP-88S5DSQ" name="features.signup.Read data from socket" skipped="0" tests="4" time="3.464004" timestamp="2018-05-02T20:05:45.408781"><testcase classname="features.signup.Read data from socket" name="Run Dump on default PC and default port" status="passed" time="0.139474"><system-out>
<![CDATA[
@scenario.begin
  Scenario: Run Dump on default PC and default port
    Given Server is running on port 4444 ... passed in 0.001s
    When dumper is started with lol.txt ... passed in 0.066s
    Then we can kill server ... passed in 0.071s
    Then Output is empty ... passed in 0.000s
    Then Error is empty ... passed in 0.001s
    Then Exit code is 0 ... passed in 0.000s
    Then Expected file is created and same with lol1.txt ... passed in 0.001s

@scenario.end
--------------------------------------------------------------------------------

Captured stdout:
13140
b''
<class 'bytes'>

]]>
</system-out></testcase><testcase classname="features.signup.Read data from socket" name="Run Dump on default PC and not default port (server runs on other port)" status="passed" time="1.126841"><system-out>
<![CDATA[
@scenario.begin
  Scenario: Run Dump on default PC and not default port (server runs on other port)
    Given Server is running on port 4444 ... passed in 0.002s
    When dumper is started: lol3.txt and 8888 ... passed in 1.072s
    Then we can kill server ... passed in 0.053s
    Then Refused Error appears ... passed in 0.000s

@scenario.end
--------------------------------------------------------------------------------

Captured stdout:
15160
ERROR:root:[WinError 10061] No connection could be made because the target machine actively refused it


]]>
</system-out></testcase><testcase classname="features.signup.Read data from socket" name="Server is not started run dumper" status="passed" time="1.077062"><system-out>
<![CDATA[
@scenario.begin
  Scenario: Server is not started run dumper
    When dumper is started with lol4.txt ... passed in 1.076s
    Then Refused Error appears ... passed in 0.001s

@scenario.end
--------------------------------------------------------------------------------

Captured stdout:
ERROR:root:[WinError 10061] No connection could be made because the target machine actively refused it


]]>
</system-out></testcase><testcase classname="features.signup.Read data from socket" name="Run Dump with 3" status="passed" time="1.120628"><system-out>
<![CDATA[
@scenario.begin
  Scenario: Run Dump with 3
    Given Server is running on port 4444 ... passed in 0.004s
    When dumper is started: lol3.txt and 8888 ... passed in 1.067s
    Then we can kill server ... passed in 0.050s
    Then Refused Error appears ... passed in 0.000s

@scenario.end
--------------------------------------------------------------------------------

Captured stdout:
12688
ERROR:root:[WinError 10061] No connection could be made because the target machine actively refused it


]]>
</system-out></testcase></testsuite>